account-kit/java/signer-sdk/src/main/java/com/alchemy/aa/core/TekManager.java

```diff
@@ -1,48 +1,50 @@
 package com.alchemy.aa.core;
 
-import com.alchemy.aa.core.exceptions.NoTEKException;
+import com.alchemy.aa.core.exceptions.NoTekException;
+import com.google.crypto.tink.CleartextKeysetHandle;
 import com.google.crypto.tink.InsecureSecretKeyAccess;
 import com.google.crypto.tink.KeyTemplate;
 import com.google.crypto.tink.KeysetHandle;
 import com.google.crypto.tink.TinkJsonProtoKeysetFormat;
 import com.google.crypto.tink.hybrid.HpkeParameters;
 import com.google.crypto.tink.hybrid.HpkePrivateKey;
 import com.google.crypto.tink.hybrid.HpkePublicKey;
-import com.google.crypto.tink.CleartextKeysetHandle;
 import com.google.crypto.tink.hybrid.internal.HpkeContext;
 import com.google.crypto.tink.hybrid.internal.HpkeKemKeyFactory;
 import com.google.crypto.tink.hybrid.internal.HpkePrimitiveFactory;
-
 import com.google.crypto.tink.proto.KeyData;
 import com.google.crypto.tink.proto.Keyset;
 import com.google.crypto.tink.util.Bytes;
 import com.google.crypto.tink.util.SecretBytes;
 import com.google.protobuf.InvalidProtocolBufferException;
 import java.security.GeneralSecurityException;
 
-public class TEKManager {
+public class TekManager {
 
-    public TEKManager() throws GeneralSecurityException {
-    }
-
-    public static TEKManager InitializeTEKManagerFromHpkeKey(HpkePrivateKey privateKey)
-            throws GeneralSecurityException, InvalidProtocolBufferException {
+    public static TekManager initializeTekManagerFromHpkeKey(HpkePrivateKey privaTekey)
+            throws GeneralSecurityException {
         KeysetHandle keysetHandle = KeysetHandle.newBuilder()
-                .addEntry(KeysetHandle.importKey(privateKey).makePrimary().withFixedId(0)).build();
-        return InitializeTEKManagerFromKeySetHandle(keysetHandle);
+                .addEntry(KeysetHandle.importKey(privaTekey).makePrimary().withFixedId(0)).build();
+        return initializeTekManagerFromKeySetHandle(keysetHandle);
     }
 
-    public static TEKManager InitializeTEKManagerFromKeySetHandle(KeysetHandle keysetHandle)
+    public static TekManager initializeTekManagerFromKeySetHandle(KeysetHandle keysetHandle)
             throws GeneralSecurityException {
-        TEKManager tek = new TEKManager();
+        TekManager Tek = new TekManager();
 
         String serializedKeyset = TinkJsonProtoKeysetFormat.serializeKeyset(keysetHandle,
                 InsecureSecretKeyAccess.get());
-        tek.serializedKeyset = serializedKeyset.toCharArray();
-        return tek;
+        Tek.serializedKeyset = serializedKeyset.toCharArray();
+        return Tek;
     }
 
-    public HpkePublicKey createTEK() throws GeneralSecurityException, InvalidProtocolBufferException {
+    public static TekManager initializeTekManager() throws GeneralSecurityException, InvalidProtocolBufferException {
```

Comment:
Lets rename these factory methods to something like:
TekManager.fromPrivateKey
TekManager.fromKeyset
TekManager.generateNew

and make constructor private so that if we have an instance for TekManager, it's always initialized?